migration-roadmap/search.md

```diff
@@ -154,125 +201,234 @@
+| ------------------------------------------------------------------------------------- | -------------------------- | ---------------------------- |
+| `.spectrum-Search--sizeS`, `.spectrum-Search--sizeL`, `.spectrum-Search--sizeXL`      | `size`                     | Implemented                  |
+| `.spectrum-Search.is-disabled`                                                        | `disabled`                 | Implemented                  |
+| `.spectrum-Search .spectrum-HelpText`                                                 | `help-text` slot           | Implemented                  |
+| `.spectrum-Search:lang(ja)`, `.spectrum-Search:lang(ko)`, `.spectrum-Search:lang(zh)` | Language-specific styling  | Implemented                  |
+| `.spectrum-Search .spectrum-Search-clearButton`                                       | Clear button functionality | Implemented                  |
+| `.spectrum-Search .spectrum-Search-textfield`                                         | Textfield functionality    | Implemented                  |
+| `.spectrum-Search .spectrum-Search-icon`                                              | Search icon                | Implemented                  |
+| `.spectrum-Search .spectrum-Search-input`                                             | Input element              | Implemented                  |
+|                                                                                       | `action`                   | Missing from CSS             |
+|                                                                                       | `method`                   | Missing from CSS             |
+|                                                                                       | `holdValueOnEscape`        | Missing from CSS             |
+|                                                                                       | `negative-help-text` slot  | Missing from CSS             |
+| `.spectrum-Search.is-collapsed`                                                       |                            | Missing from WC (new for S2) |
+| `.spectrum-Search.is-expanded`                                                        |                            | Missing from WC (new for S2) |
 
-**Features Being Deprecated/Removed:**
+## Summary of changes
 
-- None identified - component maintains backward compatibility
+### CSS => SWC implementation gaps
 
-### Action Items for Web Component Maintainers
+The primary implementation gap between CSS and SWC is the S2 implementation of the collapsed Search variant, which allows for expansion and collapse of the search field to and from a quiet action button that uses the same magnifying glass icon. In CSS, these collapsed/expanded states and controlled by the classes `.spectrum-Search.is-collapsed` and `.spectrum-Search.is-expanded`).
```

Comment:
I found a stray parentheses! What's a singular parenthesis? Apparently neither triggers spell check for me... 😆